src/main/java/io/jenkins/plugins/audit/listeners/ItemChangeListener.java

```diff
@@ -0,0 +1,95 @@
+package io.jenkins.plugins.audit.listeners;
+
+import hudson.Extension;
+import hudson.ExtensionList;
+import hudson.model.Item;
+import io.jenkins.plugins.audit.event.*;
+import org.apache.logging.log4j.audit.LogEventFactory;
+import static io.jenkins.plugins.audit.helpers.DateTimeHelper.formatDateISO;
+
+import javax.annotation.Nonnull;
+import java.time.Instant;
+
+/**
+ * Listener notified of any CRUD operations on items.
+ */
+@Extension
+public class ItemChangeListener extends hudson.model.listeners.ItemListener{
+
+    /**
+     * Fired when a new job is created and added to Jenkins, before the initial configuration page is provided.
+     *
+     * @param item is a job that is created
+     */
+    @Override
+    public void onCreated(@Nonnull Item item) {
+        CreateItem itemCreateEvent = LogEventFactory.getEvent(CreateItem.class);
+
+        itemCreateEvent.setItemName(item.getName());
+        itemCreateEvent.setItemUri(item.getUrl());
+        itemCreateEvent.setTimestamp(formatDateISO(Instant.now().toEpochMilli()));
```

Comment:
```suggestion
        itemCreateEvent.setTimestamp(formatDateISO(System.currentTimeMillis()));
```

This would be the most direct way to get the timestamp.